### PR TITLE
[luci] Verify quantization of Concat

### PR DESCRIPTION
**Pull request by jinevening**
This verifies quantization of Concat op.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: #6367

### CODE REVIEW

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by dayo09 — DISMISSED** [review]
LGTM

File: compiler/luci/pass/src/QuantizedModelVerifier.test.cpp [hunk 1]
```diff
@@ -367,6 +367,36 @@ class TransposeTestGraph final : public luci::test::TestIOGraph
   luci::CircleConst *_perm = nullptr;
 };
 
+class ConcatenationTestGraph final : public luci::test::TestIOGraph
+{
+public:
+  void init(void)
+  {
+    TestIOGraph::init({16}, {32});
+    _param = g()->nodes()->create<luci::CircleConst>();
```
Comment (seanshpark) [inline compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:376]: maybe use `create_dummy_const()` ?

Comment (jinevening) [reply compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:376]: That seems style change. I'll update it with other graphs as well.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by dayo09 — APPROVED** [review]
LGTM

**Pull request merged**